app/controllers/github_auth_controller.rb

```diff
@@ -46,7 +46,9 @@ def delete_application_grant
     Octokit.client_id = ENV['GH_AUTH_ID']
     Octokit.client_secret = ENV['GH_AUTH_SECRET']
     Octokit.client.as_app do |client|
-      client.delete "/applications/#{ENV['GH_AUTH_ID']}/grants/#{github_session.token}"
+      client.delete("/applications/#{ENV['GH_AUTH_ID']}/grant", {
+        access_token: github_session.token
+      })
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
Layout/FirstHashElementIndentation: Indent the right brace the same as the first position after the preceding left parenthesis.